rmgpy/rmg/model.py

```diff
@@ -858,6 +859,13 @@ def apply_kinetics_to_reaction(self, reaction):
                 reaction.reverse = None
         reaction.kinetics = kinetics
 
+        if isinstance(kinetics, (SurfaceArrhenius, StickingCoefficient, SurfaceArrheniusBEP, StickingCoefficientBEP)):
+            if kinetics.coverage_dependence:
+                for species, values in kinetics.coverage_dependence.items():
+                    species_in_model = self.make_new_species(species)
+                    kinetics.coverage_dependence[species_in_model] = values
+                    kinetics.coverage_dependence.remove(species)
```

Comment:
`pop` instead of `remove`? Although not sure about removing and adding things to dict as we iterate through it